unni.py

```diff
@@ -148,6 +148,8 @@ def get_job(jobs, chat_id):
 
 
 def sub_job_handle(bot, job):
+    bot.send_message(chat_id=job.context.message.chat_id, text=config.get('messages', 'subscribe_header')
```

Comment:
get the subscribe_header first and put it in a variable. see how we do it elsewhere.

it will make the code easier to read